editor/src/main/com/mbrlabs/mundus/editor/ui/modules/dock/assets/AssetsDock.kt

```diff
@@ -70,29 +65,18 @@ class AssetsDock : Tab(false, false),
     private val assetItems = Array<AssetItem>()
 
     private val assetOpsMenu = PopupMenu()
-    private val renameAsset = MenuItem("Rename Asset")
+    //        TODO: figure out if this did anything
```

Comment:
Did the rename option do anything before?

Reply:
As I see this feature has not implemented yet. Good catch. Could you add back this button? I've created an issue for it: https://github.com/JamesTKhan/Mundus/issues/279, so it won't be forgotten.